data/zilliqa/assets/zil1nfuy0fuvdun80m6f5vxfqycz0caphpf8tksmlt/meta.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "Dope Crew Friends",
+	"symbol": "DCF",
+	"links": {
+		"github": "dopecrewfriends",
```

Comment:
nothing on the github, remove.

Reply:
Removed github, added web + insta + telegram.